assemblyline/readme.md

```diff
@@ -25,6 +25,7 @@ Please see: https://cybercentrecanada.github.io/assemblyline4_docs/installation/
 | `assemblylineUIImage`            | Container image name for API server component.                                                                                                     | `cccs/assemblyline-ui`                                                  |
 | `autoInstallServices`            | A list of services to install on `helm install`. Used in conjuction with `assemblylineServiceImagePrefix`.                                         | See: [Values.yaml](./values.yaml)                                       |
 | `configuration`                  | [Assemblyline's Configuration](https://cybercentrecanada.github.io/assemblyline4_docs/odm/models/config/) Block                                    | See: [Values.yaml](./values.yaml)                                       |
+| `coreAffinity`                   | Affinity applied to all core pods.                                   | `null`
```

Comment:
Did you mean nodeAffinity here or should all the others be changed to coreAffinity?